pkg/serverless/proc/clock_unix.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build !windows
+
+package proc
+
+import "github.com/tklauser/go-sysconf"
```

Comment:
this may need to go in our [license.csv](https://github.com/DataDog/datadog-agent/blob/main/LICENSE-3rdparty.csv) file, if it's not already there.

Reply:
sounds good, confirmed it's there: https://github.com/DataDog/datadog-agent/blob/6b2b5a34201e17b2db43b29bfb9ae1adc196afe4/LICENSE-3rdparty.csv#L2796